pages.de/common/pcapfix.md

```diff
@@ -0,0 +1,24 @@
+# pcapfix
+
+> Repariert beschädigte oder korrupte `pcap`- und `pcapng`-Dateien.
+> Weitere Informationen: <https://f00l.de/pcapfix/>.
+
+- Repariert eine `pcap`/`pcapng`-Datei (Hinweis: bei `pcap`-Dateien werden nur die ersten 262144 Bytes jedes Pakets gescannt):
+
+`pcapfix {{pfad/zu/datei.pcapng}}`
+
+- Repariert eine ganze `pcap`-Datei:
+
+`pcapfix --deep-scan {{pfad/zu/datei.pcap}}`
+
+- Repariert eine `pcap`/`pcapng`-Datei und schreibt die reparierte Datei an einen bestimmten Speicherort:
+
+`pcapfix --outfile {{pfad/zu/repariert.pcap}} {{pfad/zu/datei.pcap}}`
+
+- Repariert eine `pcapng`-Datei und behandelt diese als `pcapng`-Datei, unabhängig von der automatischen Erkennung:
```

Comment:
```suggestion
- Behandle die zu reparierende Datei als `pcapng`-Datei, unabhängig von der automatischen Typenerkennung:
```

Reply:
But that wouldn't be a translation of the English TLDR page, would it?

Reply:
It would not be a 1:1 translation anymore, that's right. However, it would be a more more natural German sentence that still describes the example adequately (and arguably better than the English version). 

I personally would opt for applying my suggestion and changing the english description to something like: "Treat the specified file as a `pcapng` file, ignoring automatic recognition"

Reply:
If two people think that's better, I'm probably outvoted.